src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -114,6 +117,16 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     case AdminDeleteAllRefreshTokens => asAdmin { deleteAllRefreshTokens() } pipeTo sender
   }
 
+  def asOwner(projectName: RawlsBillingProjectName)(op: => Future[PerRequestMessage]): Future[PerRequestMessage] = {
```

Comment:
Can we rename either this or `asAdmin`? Firecloud admin is not the same kind of role as project owner/user. Maybe `asFCAdmin` and/or `asProjectOwner`?